optaplanner-wb-webapp/.gitignore

```diff
@@ -1,10 +1,9 @@
 /target
 /local
 /repositories
-/src/main/gwt-unitCache/
```

Comment:
Ah, seems we already did this in the past - and now we do it less - so this PR is a step forward that that previous comment should not prevent the merge. It is an open issue though.